lib/Zonemaster/Backend/DB/MySQL.pm

```diff
@@ -104,7 +104,7 @@ sub create_db {
     # TEST RESULTS
     ####################################################################
     $dbh->do(
-        'CREATE TABLE test_results (
+        'CREATE TABLE IF NOT EXISTS test_results (
```

Comment:
Couldn't it be enough to check if the table exists, and if not create all tables, triggers and indexes? Could that simplify the code?

Reply:
I'm not sure. I find it better and proper to do all the checks. Then we are are sure we won't end with an improper behavior. Unless there is a better way to do all these checks, I'd like to keep them here.
And to add another argument, the goal I'd like to reach is to make it possible to let the agent call the `create_db` method on startup. So that we have a proper database on first run (without letting the user do this job for us). This would even allow to include the migration script within the code and run them on startup (but this will be done in a future work).